plugins/scenario/utils/hreForBase.ts

```diff
@@ -58,7 +58,7 @@ function hreForBase(base: ForkSpec): HardhatRuntimeEnvironment {
 
   const hardhatArguments = getEnvHardhatArguments(HARDHAT_PARAM_DEFINITIONS, process.env);
 
-  const config = loadConfigAndTasks(hardhatArguments);
+  const { resolvedConfig: config } = loadConfigAndTasks(hardhatArguments);
```

Comment:
what's the difference between `config` and `resolvedConfig`?

Reply:
They added a resolved config and user config

Reply:
This was from my bump to 2.9 PR which got merged into this one (for `mineBlocks`)